raft/rafttest/node.go

```diff
@@ -62,6 +64,22 @@ func (n *node) start() {
 				n.Node = nil
 				close(n.stopc)
 				return
+			case p := <-n.pausec:
+				recvms := make([]raftpb.Message, 0)
+				for p {
+					// TODO: locking around paused?
```

Comment:
If anything uses node.isPaused, then this will need to be locked. But nothing's using it now and I'm not sure if we even need that method (and field) at all.